src/screens/index.js

```diff
@@ -0,0 +1,3 @@
+import { LandingPage } from "./Landing";
+
+export { LandingPage };
```

Comment:
Similarly here also you can directly export instead of importing and then exporting
`export {LandingPage} from "./Landing"`

Reply:
okkk roshan will make these changes!! thankyou so much for reviewing!!